sunrisemvmtnyc/legislation-tracker
# feat: Use actual legislator data in legislator filter

Legislator filter has legislators for current bills

## To test:
1. Hit legislator dropdown & make sure it has legislators
2. Choose one legislator & make sure the shown bills have that legislator as a _sponsor_ (it shows this on the bill page)
3. Choose a second legislator & all bills should have one of the two legislators as a _sponsor_
